src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colors colors;
+    private int number;
+
+    public Ball(Colors colors, int number) {
+        this.colors = colors;
+        this.number = number;
+    }
+
+    public Colors getColors() {
+        return colors;
+    }
+
+    public void setColors(Colors colors) {
+        this.colors = colors;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    enum Colors {
+        RED("red"), ORANGE("orange"), YELLOW("yellow"),
+        GREEN("green"), BLUE("blue"), VIOLET("violet");
+        private String color;
+
+        Colors(String color) {
+            this.color = color;
+        }
+
+    }
```

Comment:
do we really need this things?
can we write just `enum Colors {RED, GREEN, BLUE}`?